serverless/eventing/channels/connecting-channels-sinks.adoc

```diff
@@ -0,0 +1,17 @@
+:_content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+[id="connecting-channels-sinks"]
+= Connecting channels to sinks
+:context: connecting-channels-sinks
+toc::[]
+
+After events have been sent to a channel from an event source or producer, these events can be sent to a sink by using a subscription. Subscriptions are created by configuring a `Subscription` object, which specifies the channel and the sink (also known as a _subscriber_) to deliver events to.
```

Comment:
```suggestion
Before events can be sent to a channel from an event source or producer, these events can be sent to a sink by using a subscription. Subscriptions are created by configuring a `Subscription` object, which specifies the channel and the sink (also known as a _subscriber_) to deliver events to.
```

We shouldn't suggest people should send data to the not-fully configured (linked) event mesh. All connections should be made first, before sending production data. Of course, users might use channels with persistence (like Kafka), but still, we shouldn't suggest sending data to not fully connected system.

Reply:
@pierDipi this was part of the reason I suggested moving these topics to group them as more of a dependency on each other - it seemed odd to me to send events to the channel without subscribers even if the idea is for things to be decoupled.

I think this user story and which one is appropriate may need further discussion within the eventing + QE teams to decide which message we want to send to users? Just changing this sentence will make this page not make sense, it won't help to clarify that we should only send events to a channel if there are subscribers.